typed/reconcile_schema.go

```diff
@@ -229,6 +222,13 @@ func (v *reconcileWithSchemaWalker) visitMapItems(t *schema.Map, element *fieldp
 }
 
 func (v *reconcileWithSchemaWalker) doMap(t *schema.Map) (errs ValidationErrors) {
+	// We don't reconcile deduced types (unstructured CRDs) or maps that contain only unknown
+	// fields. Deduced types do not support atomic or granular tags. Nor does the dynamic
```

Comment:
Can you change this wording to make it clear it's not necessarily the end of the story for deduced types, like maybe we'll fix it? (I know this was a copy-paste, sorry)

Reply:
Sure, I've rephrased it.